rfcs/text/0044-add-apple-platform-specific-fields.md

```diff
@@ -0,0 +1,119 @@
+# 0000: Name of RFC
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+
+### Summary
+This RFC proposes the addition of Apple platform-specific fields to the ECS schema. This enhancement will enable security software vendors to more accurately map out data, particularly for Apple platforms.
+
+The following feelds needs to be considered being added:
+
+## Fields
+
+##### Proposed New Fields for Process object
+
+Field | Type | Example | Description
+--- | --- | --- | ---
+responsible	| keyword	| Terminal.app	| The responsible process on macOS, from an ancestry perspective, is the process that originally launched or spawned a given process.
```

Comment:
I think the type would be process, not keyword. Do I understand correctly this should be a nested process under the field `responsible`?

Reply:
Also how it differs from parent? https://www.elastic.co/guide/en/ecs/current/ecs-process.html#ecs-process-nestings

Reply:
@trisch-me yes it should be nested under `process` which will become `process.responsible.pid` as example.

MacOS keeps running track of an additional field called responsible process. This process ties to back to the highest level application in the tree and carries TCC permissions into each child below it.